lib/ansible/modules/cloud/ovirt/ovirt_templates.py

```diff
@@ -154,6 +173,44 @@
   cluster: mycluster
   id: 1111-1111-1111-1111
 
+# Register template with vnic profile mappings
+- ovirt_templates:
+  state: registered
+  storage_domain: mystorage
+  cluster: mycluster
+  id: 1111-1111-1111-1111
+  vnic_profile_mappings:
+    - source_network_name: mynetwork
+      source_profile_name: mynetwork
+      target_profile:
+        name: target_network
+    - source_network_name: mynetwork2
+      source_profile_name: mynetwork2
+      target_profile:
+        name: target_network2
+
+# Register template with mapping
+- ovirt_templates:
+    state: registered
+    storage_domain: mystorage
+    cluster: mycluster
+    id: 1111-1111-1111-1111
+    role_mappings:
+      - source:
```

Comment:
Any reason to not use `source_name`, `dest_name`?

Reply:
Not any special reason, I will change all of the mappings to use only name